rfcs/helm-oci/README.md

```diff
@@ -86,9 +89,54 @@ spec:
       interval: 1m # check for new OCI artifacts every minute
 ```
 
+#### Story 2
+
+> As a platform admin I want to automate Helm chart updates based on a semver ranges.
+> When a new patch version is available in the container registry, I want Flux to open a PR
+> with the version set in the `HelmRelease` manifests.
+
+Given that charts are stored in container registries, you can use Flux image automation
+and patch the chart version in Git, in the same way Flux works for updating container image tags.
+
+Define an image policy using semver:
+
+```yaml
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: my-app
+  namespace: default
+spec:
+  imageRepositoryRef:
+    name: my-app
+  policy:
+    semver:
+      range: 1.0.x
+```
+
+Then add the policy marker to the `HelmRelease` manifests in Git:
+
+```yaml
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: podinfo
+  namespace: default
+spec:
+  interval: 60m
+  chart:
+    spec:
+      chart: my-app
+      version: 1.0.0 # {"$imagepolicy": "default:my-app:tag"}
+      sourceRef:
+        kind: HelmRepository
+        name: ghcr-charts
+      interval: 1m
+```
+
 ### Alternatives
 
-We could use introduce a new API type e.g. `HelmOCIRepository`. That is considered unpractical,
+We could introduce a new API type e.g. `HelmOCIRepository`. That is considered unpractical,
```

Comment:
I would at least expect an outline of the rejected previous structure, in which an `OCIRegistry` would be introduced, to be included here.

Reply:
Add a link to the previous RFC and outlined the proposal there.